R/container/README.md

```diff
@@ -33,22 +32,23 @@ container_pull(
 ```
 
 <a name="r_library_image"></a>
+
 ## r_library_image
 
 In a `BUILD` file, define your library of R packages and install them
 in a Docker image using this rule. Dependencies are installed implicitly
 in an efficient layered mechanism.
 
 ```python
-load("@com_grail_rules_r//R:defs.bzl", "r_library")
+load("@com_rules_r//R:defs.bzl", "r_library")
```

Comment:
Can we rename these to `rules_r` instead?

Reply:
That's fine with me!